Bibliotheque.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Ipssi\Evaluation;
+
+
+/**
+ * 
+
+ */
+class Bibliotheque
+{
+
+	private $adherents = array();
+	private $oeuvres = array();
+
+	function __construct( $oeuvre, $adherent)
```

Comment:
```suggestion
	function __construct(array $oeuvre, array $adherent)
```
Idem il faut vérifier le contenu du tableau